### ISSUE TITLE
[Need to Triage\Review Meeting][Web] [Accessibility] [A11yMAS][Name, Role and Value] [MAS4.1.2] [State(Expanded\Collapsed) is not announced for Explore Menu when Menu is Expanded and Collapsed.]

### ISSUE THREAD

**Issue by DasaradhaKamjula** [OfficeDev/script-lab#678]
**User Experience:**
User who rely on AT will not get proper information whether “Menu” is expanded or collapsed if  screen reader is not providing clear information about the State of control.
 
**Test Environment:**
OS Build: 16299.19
Browser: Microsoft Edge 41.16299.15.0
Tools used: Narrator
**Attachment:**
[MAS4.1.2_State is announced.zip](https://github.com/OfficeDev/script-lab/files/1440404/MAS4.1.2_State.is.announced.zip)

**Repro Steps:**
1. Open the URL: http://scriptlabdocsdemo.azurewebsites.net/Office%20Dev%20Center%20-%20Docs%20-%20Build%20your%20first%20Excel%20add-in.html .
2. Navigate to Explore Menu and press Enter.
3. Verify whether state is announced by screen reader when Explore menu is Expanded or collapsed. 
 
**Current Behavior:** 
State(Expanded\Collapsed) is not announced for Explore Menu when Menu is Expanded and Collapsed.

**Expected Behavior:**
State(Expanded\Collapsed) should be announced by screen reader for Explore Menu when Menu is Expanded and Collapsed. So that user will get to know whether Menu is opened or closed.

**Note:** 
Same Issue is reproducible for ”Resources” and “Documentation” Menu controls in the Header. State is not announced for those Menu controls when user expands and close the Menu

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}
 
**Suggested Fix:**
https://microsoft.sharepoint.com/teams/msenable/mas/pages/mas-details.aspx?masid=4.1.2
